actions/upgrade-containerd

```diff
@@ -12,23 +12,28 @@ from charmhelpers.fetch import (
     apt_unhold
 )
 
-from charmhelpers.core import host
+from charmhelpers.core.host import service_restart
+
+from charms.reactive import remove_state
+
+from reactive.containerd import CONTAINERD_PACKAGE
```

Comment:
Does this import work? I've been under the impression that the charm's `reactive` folder isn't included in the python path and we can't import from it. Usually if something needs to be imported from elsewhere, we put it in the charm's `lib` folder instead.

Reply:
It's worked fine in my tests (deployed and ran the run action with `--wait`)

```
juju run-action --wait containerd/0 upgrade-containerd        
unit-containerd-0:
  UnitId: containerd/0
  id: "3"
  results:
    Stdout: |
      Hit:1 http://eu-central-1.ec2.archive.ubuntu.com/ubuntu bionic InRelease
      Hit:2 http://eu-central-1.ec2.archive.ubuntu.com/ubuntu bionic-updates InRelease
      Hit:3 http://eu-central-1.ec2.archive.ubuntu.com/ubuntu bionic-backports InRelease
      Hit:4 http://security.ubuntu.com/ubuntu bionic-security InRelease
      Reading package lists...
      Canceled hold on containerd.
      Reading package lists...
      Building dependency tree...
      Reading state information...
      containerd is already the newest version (1.3.3-0ubuntu1~18.04.1).
      0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
      containerd set on hold.
    runtime: containerd
  status: completed
  timing:
    completed: 2020-03-04 16:54:14 +0000 UTC
    enqueued: 2020-03-04 16:54:08 +0000 UTC
    started: 2020-03-04 16:54:10 +0000 UTC

```

Reply:
I have no issues with putting just this in `./lib/` if it's best practice though.

Reply:
Ahh cool. Nah, if it works then I'm content. Thanks!